<!--REQUEST_ID=3ac8aa3d-c561-49ab-8cb6-da7dd619db71-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50284&attempt_id=335c8fa5-38c1-4260-a006-652f4c7026cd)

Attempted: hmmer

<details><summary>Partial log (click to expand)</summary><p>

```
checking whether more special flags are required for pthreads... no
checking for PTHREAD_PRIO_INHERIT... yes
checking whether gcc can compile our Altivec/VMX code... no
checking whether gcc can compile our SSE code... no
configure: ::::::::::--- no vector instruction set ---::::::::::
configure: HMMER3 requires SSE or VMX vector instructions.
configure: Supported platforms are x86 (Intel/AMD) and PowerPC.
configure: error: No supported vectorization found for your machine.
builder for '/nix/store/mcg6gmhqlhys3wl8b5i6w94bq5mhdw9x-hmmer-3.2.1.drv' failed with exit code 1
error: build of '/nix/store/mcg6gmhqlhys3wl8b5i6w94bq5mhdw9x-hmmer-3.2.1.drv' failed
```
</p></details>

--- a/pkgs/applications/science/biology/hmmer/default.nix
+++ b/pkgs/applications/science/biology/hmmer/default.nix
@@ -1,28 +1,25 @@
 { stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
-  version = "3.1b2";
+  version = "3.2.1";
   name = "hmmer-${version}";
 
   src = fetchurl {
-    url = "http://eddylab.org/software/hmmer3/${version}/${name}.tar.gz";
-    sha256 = "0djmgc0pfli0jilfx8hql1axhwhqxqb8rxg2r5rg07aw73sfs5nx";
+    url = "http://eddylab.org/software/hmmer/${name}.tar.gz";
+    sha256 = "171bivy6xhgjsz5nv53n81pc3frnwz29ylblawk2bv46szwjjqd5";
   };
 
   meta = with stdenv.lib; {
     description = "Biosequence analysis using profile hidden Markov models";
     longDescription = ''
-HMMER is used for searching sequence databases for sequence homologs, and for making sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs).
-
-HMMER is often used together with a profile database, such as Pfam or many of the databases that participate in Interpro. But HMMER can also work with query sequences, not just profiles, just like BLAST. For example, you can search a protein query sequence against a database with phmmer, or do an iterative search with jackhmmer.
-
-HMMER is designed to detect remote homologs as sensitively as possible, relying on the strength of its underlying probability models. In the past, this strength came at significant computational expense, but as of the new HMMER3 project, HMMER is now essentially as fast as BLAST.
-
-HMMER can be downloaded and installed as a command line tool on your own hardware, and now it is also more widely accessible to the scientific community via new search servers at the European Bioinformatics Institute.
+      HMMER is used for searching sequence databases for sequence homologs, and for making sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs).
+      HMMER is often used together with a profile database, such as Pfam or many of the databases that participate in Interpro. But HMMER can also work with query sequences, not just profiles, just like BLAST. For example, you can search a protein query sequence against a database with phmmer, or do an iterative search with jackhmmer.
+      HMMER is designed to detect remote homologs as sensitively as possible, relying on the strength of its underlying probability models. In the past, this strength came at significant computational expense, but as of the new HMMER3 project, HMMER is now essentially as fast as BLAST.
+      HMMER can be downloaded and installed as a command line tool on your own hardware, and now it is also more widely accessible to the scientific community via new search servers at the European Bioinformatics Institute.
     '';
     homepage = http://hmmer.org/;
     license = licenses.gpl3;
     maintainers = [ maintainers.iimog ];
-    platforms = platforms.linux;
+    platforms = [ "x86_64-linux" "i686-linux" ];
   };
 }

